### ISSUE TITLE
Feature: use drone build as a trigger

### ISSUE THREAD

**Issue by armand1m** [Odusanya18/droneci-spinnaker-stage#5]
Hey! Thank you a lot for contributing to spinnaker and introducing this plugin.

I am deploying it into a cluster of mine and am wondering if you plan to add support to use a Drone Build as a Trigger for a pipeline. If so, I'd be glad to try and contribute with a PR for that!

**Comment by Odusanya18** [comment]
Yes, I did intend to add DroneCI Build Trigger support, but I don't have much bandwidth to take it on. Gladly! That'd be great @armand1m !

**Comment by armand1m** [comment]
@Odusanya18 nice, I'll try to get started with it by this week then. Will keep you posted in the PR once opened.
